ecosystem/sep-0011.md

```diff
@@ -170,19 +170,34 @@ Enums are represented by the bare keyword of the value.  They can also be specif
 
 A few aggregate values are special-cased:
 
-##### Asset, TrustLineAsset, ChangeTrustAsset
+##### AlphaNum4, AlphaNum12
 
-The `Asset`, `TrustLineAsset`, and `ChangeTrustAsset` union types are rendered as a string in one of three forms dependent on their discriminant:
+The `AlphaNum4` and `AlphaNum12` struct types are both rendered as a string
+`Code:IssuerAccountID`.
 
-  - `native` (or any string up to 12 characters not containing an unescaped colon) for the native asset, i.e. `ASSET_TYPE_NATIVE`.
+"Code" must consist of printable ASCII characters (octets 0x21 through 0x7e).
+
+The sequence `\x` introduces a hex escape sequence, e.g., `\x00` to introduce a
+0-valued byte.  Otherwise, `\` escapes the next character, so `\\` is required
+to introduce a backslash.
+
+Stellar disallows assets of type `ASSET_TYPE_CREDIT_ALPHANUM12` that have fewer
+than 5 bytes, but such assets can be represented in binary XDR, and so in txrep
+an `AlphaNum12` value with fewer than 5 bytes in its `assetCode` is rendered
+with trailing `\x00` (escaped NUL bytes) to as to make the length 5---e.g., the
+12-byte asset code `ABC` is rendered `ABC\x00\x00`. Note that stellar-core
+disallows non ASCII bytes in `AssetCode` fields, so the primary use of this
+feature is to construct or examine invalid transaction test cases.
 
-  - `Code:IssuerAccountID` for issued assets, i.e. assets of type `ASSET_TYPE_CREDIT_ALPHANUM4` and `ASSET_TYPE_CREDIT_ALPHANUM12`.
+##### Asset, TrustLineAsset
```

Comment:
Sorry for all of the back-and-forth, but since this is a formal spec we've gotta nail the details. Are we saying here that `ChangeTrustAsset` will not have a SEP-11 form at all, even if it contains a native/credit4/credit12 asset? Seems fine, but we should be explicit. Alternatively, a subsection for the LPs where we say "the following does not apply to ChangeTrustAsset".

Reply:
> This section only details exceptions the standard format

This is the knowledge I was missing! Thanks for cluing me in, and definitely appreciate the detailed explanation :100: all of my concerns have melted away :ok_hand: